setup.py

```diff
@@ -47,7 +47,7 @@
         "docker",
         "graphviz",
         "loguru",
-        "layered_config_tree",
+        "layered_config_tree>=3.0.0",
```

Comment:
Required b/c this version properly handles dunder-like attrs (required by deepcopy)